### PR TITLE
Made event post for techtalk with Monica Beckwith

### PR DESCRIPTION
**Pull request by fat1996**

### CODE REVIEW

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
++Title: Techtalk with Monica Beckwith
```
Comment (evykassirer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: it's not showing up on the test site because of these `+`s at the start of your lines

**Comment by fat1996** [comment]
Removed the extra +'s.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md (hunk 1 above)
Comment (fat1996) [reply content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: Removed the extra +'s.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: Maybe something like "Monica Beckwith will be visiting campus to give a talk on her work in ...". You can have a look at https://github.com/wics-uw/website/blob/master/content/W2015/W15-event-hypatia.md for inspiration.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Title: Techtalk with Monica Beckwith
```
Comment (evykassirer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: I think it should be two words "tech talk"

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
+
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade not just optimizing the JVM heuristics, but also improving the Just-in-time(JIT) code quality for various processor architectures as well as working with the garbage collectors and improving garbage collection for server systems.
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: 80 chars per line.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she will dive into the HotSpot garbage collection and provide an overview of the various garbage collectors available in HotSpot.
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: 80 chars per line.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md (hunk 4 above)
Comment (evykassirer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: lines should be max 80 characters

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## Event Details ##
+
++ **Who:** This is a women-only event.
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: Need to link to FAQ. Edit: Oh actually this is open to everyone!

Comment (evykassirer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: it's open to everyone!

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade not just optimizing the JVM heuristics, but also improving the Just-in-time(JIT) code quality for various processor architectures as well as working with the garbage collectors and improving garbage collection for server systems.
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she will dive into the HotSpot garbage collection and provide an overview of the various garbage collectors available in HotSpot.
+
+## Event Details ##
+
++ **Who:** This is a women-only event. 
++ **What:** Come join us for a talk with Monica Beckwith
++ **Where**: STC 0040
++ **When:** Tuesday March 14, 6:00-7.30pm
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: 7.30pm -> 7:30pm

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+Summary: Come join us at a tech talk with Monica Beckwith!
+
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade not just optimizing the JVM heuristics, but also improving the Just-in-time(JIT) code quality for various processor architectures as well as working with the garbage collectors and improving garbage collection for server systems.
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she will dive into the HotSpot garbage collection and provide an overview of the various garbage collectors available in HotSpot.
+
+## Event Details ##
+
++ **Who:** This is a women-only event. 
++ **What:** Come join us for a talk with Monica Beckwith
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: This should be the name of the talk

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
```
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: This is a little long now 😛

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md (hunk 3 above)
Comment (annalorimer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md]: Techtalk -> Tech Talk

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Review by evykassirer — APPROVED** [review]

**Pull request merged**